php/exceptions/NotFound.class.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * This class is an Exception extension created to distinguish any
+ * Exception from the case when a module can't be loaded because it is not a module
+ * (instruments, forms, reliability forms,...)
+ *
```

Comment:
should have an ending dot after the parenthesis

Reply:
took me a minute to parse what you meant, but I think changed it..

Reply:
Will try to be more clear next time